{
  for (  Object exception : thrownExceptions) {
    mapThrownException((Name)exception,member);
  }
}
